http-conduit/Network/HTTP/Simple.hs

```diff
@@ -76,6 +76,9 @@ module Network.HTTP.Simple
     , H.setRequestIgnoreStatus
     , H.setRequestCheckStatus
     , setRequestBasicAuth
+#if MIN_VERSION_http_client(0,7,6)
```

Comment:
Using CPP for backwards compat with old http-clients matches the existing usage for `shouldStripHeaderOnRedirect`